### ISSUE TITLE
Use yaml or yml consistently. 

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/cellfinder#465]
In the code and the docs we have a mix of the two which is confusing. We should decide on one and stick to it (and then update the docs and brainmapper).

Image.sc issue for reference: https://forum.image.sc/t/yaml-files-for-network-retraining/104031
Docs issue: https://github.com/brainglobe/brainglobe.github.io/issues/255

**Comment by nischitkumar** [comment]
Hey @adamltyson , could I take up this issue?

**Comment by adamltyson** [comment]
Of course! Would you mind also tackling https://github.com/brainglobe/brainglobe.github.io/issues/255 to ensure that the code and the docs are in sync?

**Comment by adamltyson** [comment]
Closed by #488
